server/events.py

```diff
@@ -66,13 +60,7 @@ def valid_difficulty(cls, value, values):  # noqa: U100
 
 
 class DisconnectData(EventData):
-    """The data of a disconnection event.
-
-    Fields:
-        username: The username of the user disconnecting.
-    """
-
-    username: str
+    """The data of a disconnection event."""
```

Comment:
How come the username was removed? I think the frontend wanted this to remove them as a collaborator

Reply:
We are now sending a `sync` event instead, this avoids problems in case two users have the same username.

Reply:
> We are now sending a `sync` event instead, this avoids problems in case two users have the same username.

Ok, then line 78 of server/client.py needs updating